docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md

```diff
@@ -26,55 +26,57 @@ The following chapter illustrates the process from searching for an EDC point, t
-| [005](../resources/development-view/catena-x-pcf-endpoint-1_1_0.yaml) (Requesting PCF Value)                                                                                                                                                                                                                                                            | GET    | /productIds                                                                                          | {productId}                                                                                                       |
-| [006](../resources/development-view/catena-x-pcf-endpoint-1_1_0.yaml) (Sending PCF Value)                                                                                                                                                                                                                                                                 | PUT    | /productIds                                                                                          | {productId}                                                                                                       |
+| [004](https://eclipse-tractusx.github.io/docs-kits/next/kits/Digital%20Twin%20Kit/Software%20Development%20View/API%20AAS%20Registry/get-all-asset-administration-shell-descriptors) (Lookup PCF submodel/EDC asset ID)                                                                | GET    | /shell-descriptors                                                                                   | `{DIGITAL TWIN ID}`                                                                                               |
+| [005](../resources/development-view/catena-x-pcf-endpoint-1_1_1.yaml) (Requesting PCF value)                                                                                                                                                                                                                                                            | GET    | /productIds                                                                                          | {productId}                                                                                                       |
+| [006](../resources/development-view/catena-x-pcf-endpoint-1_1_1.yaml) (Sending PCF value)                                                                                                                                                                                                                                                                 | PUT    | /productIds                                                                                          | {productId}                                                                                                       |
```

Comment:
As the API plugin within Docusaurus will be discontinued in the near future and, thus, the links will stop working, please remove or update the references where possible. See: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/issues/970

Reply:
I removed the MDX files, but kept an empty sidebar.js so build does not break. To also remove this file, I would have to change the top level sidebar.js. Didn't want to change on top level without agreement up front. ;-)

In addition: Is there a break save way to link to the openAPI folder? Did not want to do this relatively and an absolute link did not work.

Reply:
Unfortunately I think, for now you should stick with the relative paths. In my understanding, these files will eventually be automatically copied and rendered with swagger. But for now I'm not sure whether it is already automated or not. Hence, we may need to fix the links later on. 

Example link: https://eclipse-tractusx.github.io/api-hub/puris/1.0.0/swagger-ui/
In this case it should look like: https://eclipse-tractusx.github.io/api-hub/pcf/1.1.1/swagger-ui/